apps/src/code-studio/pd/foorm/editor/components/FoormEntityEditorHeader.jsx

```diff
@@ -42,16 +34,19 @@ const PREVIEW_ON = 'preview-on';
 const PREVIEW_OFF = 'preview-off';
 const TIME_FORMAT = 'h:mm a';
 
-class FoormEditorHeader extends Component {
+// Metadata and buttons for the Foorm Editor. Shows name and version,
+// live preview toggle, and validate button/validation status.
+class FoormEntityEditorHeader extends Component {
   static propTypes = {
     livePreviewToggled: PropTypes.func,
     livePreviewStatus: PropTypes.string,
+    validateURL: PropTypes.string,
+    validateDataKey: PropTypes.string,
+    headerTitle: PropTypes.node,
 
     // populated by Redux
-    formQuestions: PropTypes.object,
-    isFormPublished: PropTypes.bool,
-    formName: PropTypes.string,
-    formVersion: PropTypes.number
+    questions: PropTypes.object,
+    editorType: PropTypes.string
```

Comment:
i don't see this in `connect`. also i don't see it being used anywhere. if you're going to need it in your next PR i'm fine with leaving it here, but in general we don't want to merge unused code.

Reply:
I'll fix now, good catch